SPECS/xxhash/xxhash.spec

```diff
@@ -0,0 +1,186 @@
+Name:     xxhash
+Version:  0.8.1
+Release:  3%{?dist}
+Summary:  Extremely fast hash algorithm
+#    The source for the library (xxhash.c and xxhash.h) is BSD
+#    The source for the command line tool (xxhsum.c) is GPLv2+
+License:  BSD and GPLv2+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:      http://www.xxhash.com/
+Source0:  https://github.com/Cyan4973/xxHash/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+#    Fix compilation on RHEL 7 ppc64le (gcc 4.8)
+#    https://github.com/Cyan4973/xxHash/issues/622
+#    https://github.com/Cyan4973/xxHash/pull/631
+Patch0:   %{name}-epel7-ppc64le.patch
```

Comment:
Do we need this patch? We don't support ppc64le architectures, and our GCC is a lot more recent than the RHEL 7 version.

Reply:
No, it is not needed for CBL-Mariner. Removed this patch